modules/lunamediahbBidAdapter.js

```diff
@@ -74,10 +74,13 @@ export const spec = {
       if (mediaType && mediaType[BANNER] && mediaType[BANNER].sizes) {
         placement.sizes = mediaType[BANNER].sizes;
         placement.traffic = BANNER;
-      } else if (mediaType && mediaType[VIDEO] && mediaType[VIDEO].playerSize) {
-        placement.wPlayer = mediaType[VIDEO].playerSize[0];
-        placement.hPlayer = mediaType[VIDEO].playerSize[1];
+      } else if (mediaType && mediaType[VIDEO]) {
+        if (mediaType[VIDEO].playerSize) {
+          placement.wPlayer = mediaType[VIDEO].playerSize[0];
```

Comment:
Prebid nests the playerSize array that is defined in the ad unit, meaning that [640, 480] is converted to [[640,480]]. So currently, you will only be passing an array through placement.wPlayer and placement.hPlayer is undefined and therefore dropped. I believe you are trying to pass the individual number values of the playerSize through their respective properties, so you will need to look into the second array in order to do so.

```
placement.wPlayer = mediaType[VIDEO].playerSize[0][0];
placement.hPlayer = mediaType[VIDEO].playerSize[0][1];
```